simapp/abci.go

```diff
@@ -0,0 +1,81 @@
+package simapp
```

Comment:
```suggestion
// this is a vote extension specifically for testing, please do not copy
package simapp
```

something like that or not needed?